src/utils.js

```diff
@@ -53,6 +53,15 @@ exports.defaultTitle = function () {
   return `${process.env.npm_package_name || 'Webpack Bundle Analyzer'} [${currentTime}]`;
 };
 
+exports.defaultLogInfo = function (options) {
+  const {host, port} = options;
+  const url = `http://${host}:${port}`;
+  return (
+    `${bold("Webpack Bundle Analyzer")} is started at ${bold(url)}\n` +
+    `Use ${bold("Ctrl+C")} to close it`
+  );
```

Comment:
This now duplicates the "Webpack Bundle Analyzer is started at..." text by default, as it's also written here:

https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/45b4db76dff3f3b21cea5ae7f9859c60f2cb0f7d/src/viewer.js#L82-L85

I wonder if your use case could also be implemented by only changing how the `url` is calculated? So that we would not need to allow users of this plugin to also override the "Webpack Bundle Analyzer is started" and "Use Ctrl+C to close it" texts.

Then we could change the name of this option to not be `logInfo` but something like `analyzerUrl`, and usage would look like:

```js
const url = analyzerUrl({
  listenPort: port,
  listenHost: host,
  boundAddress: server.address()
});
```

We should document that the `server.address()` is the value as returned by `http` module: https://nodejs.org/api/net.html#serveraddress

The default implementation should be what is the current returned value:

```js
exports.defaultAnalyzerUrl = function (options) {
  const { listenHost, boundAddress } = options;
  return `http://${listenHost}:${boundAddress.port}`;
}
```

Reply:
copy that! I fixed. Any other suggestions？